docs/manual/source/community/projects.html.md

```diff
@@ -29,79 +29,45 @@ and submit a pull request.
 
 ## SDKs
 
-Community-powered SDKs are [separately listed](/sdk/).
+### Swift SDK
 
-## Magento 2 Personalised Products Module
+ - Minh-Tu Le: https://github.com/minhtule/PredictionIO-Swift-SDK
 
-URL: https://github.com/richdynamix/personalised-products
 
-Personalised Products is a Magento 2 module that will serve realtime predicted suggestions for product upsells on the product page and complimentary suggestions for cross sells on the basket page. All powered by PredictionIO using the [Similar Product](/gallery/template-gallery/#recommender-systems "Similar Product") engine and the [Complementary Purchase](/gallery/template-gallery/#unsupervised-learning "Complementary Purchase") engine.
+## DEMOs
 
-- Core Author: Steven Richardson
+### Tapster iOS Demo
 
-- Status: It works with PredictionIO v0.9.5 - Under active development
+ - Minh-Tu Le: https://github.com/minhtule/Tapster-iOS-Demo
 
-## Vagrant Installation for Apache PredictionIO®
 
-URL: https://github.com/PredictionIO/PredictionIO-Vagrant
+## Template Maintainer
 
-Bring Up PredictionIO 0.9.x VM with Vagrant.
+ - Naoki Takezoe: https://github.com/takezoe/predictionio-templates-maintenance
```

Comment:
Sorry for my delayed comment. It doesn't need to credit me because I'm one of PredictionIO committers and maintain official templates under ASF is part of our responsibility. And `predictionio-templates-maintenance` repository is my temporary tools to reduce maintaining costs so it's not appropriate to be listed here.

Reply:
I think when something got made, it is fine that we list it even if it is part of the duty.
But anyway, should we open another issue on Jira and edit it out?

Reply:
In fact, templates have been maintained by PredictionIO team. I did it for just 0.13.0 release.

Anyway, I will create an another pull request because I want to do further update for the community page as well.